examples/006_filter_composite_data_example.py

```diff
@@ -139,7 +139,9 @@
         )
 
         value = stress_data[selected_indices][:, component]
-        local_result_field.append(value, element_id)
+        # value needs to be passed as list because dpf does not support numpy
+        # slices in the append call.
+        local_result_field.append(list(value), element_id)
```

Comment:
I wonder whether `.tolist()` (https://numpy.org/doc/stable/reference/generated/numpy.ndarray.tolist.html#numpy-ndarray-tolist) or `list()` is better here. Most probably, it doesn't make a huge difference.

Reply:
Good point. It is for sure better for nested arrays.